solenoid/records/tests/test_models.py

```diff
@@ -41,27 +41,43 @@ def setUp(self):
             Fields.TITLE: "Ultraviolet behavior of non-abelian gauge theories",
         }
 
+    def test_metadata_not_missing_any_citation_fields(self):
+        metadata = copy.copy(self.paper_data)
+        metadata[Fields.CITATION] = "This is a citation"
+        metadata[Fields.TITLE] = "This is a paper title"
+        metadata[Fields.JOURNAL] = "Journal of Awesomeness"
+        assert not Record.missing_citation_fields(metadata)
+
     # need to actually test create_citation
-    def test_is_metadata_valid_yes_citation_no_citation_data(self):
+    def test_is_cited_metadata_missing_minimal_citation_fields(self):
         metadata = copy.copy(self.paper_data)
         metadata[Fields.CITATION] = "This is a citation"
         metadata[Fields.TITLE] = None
         metadata[Fields.JOURNAL] = None
-        assert Record.is_data_valid(metadata)
+        assert Record.missing_citation_fields(metadata) == (
+            "Citation was not provided and/or "
+            f"missing required fields to generate minimal citation: ['{Fields.TITLE}', '{Fields.JOURNAL}']."
+        )
 
-    def test_is_metadata_valid_no_citation_yes_citation_data(self):
+    def test_uncited_metadata_with_minimal_citation_fields(self):
         metadata = copy.copy(self.paper_data)
         metadata[Fields.CITATION] = None
         metadata[Fields.TITLE] = "This is a paper title"
         metadata[Fields.JOURNAL] = "Journal of Awesomeness"
-        assert Record.is_data_valid(metadata)
+        assert Record.missing_citation_fields(metadata) == (
+            "Citation was not provided and/or "
+            f"missing required fields to generate minimal citation: ['{Fields.CITATION}']."
+        )
 
-    def test_is_metadata_valid_no_citation_no_citation_data(self):
+    def test_is_uncited_metadata_missing_citation_fields(self):
         metadata = copy.copy(self.paper_data)
         metadata[Fields.CITATION] = None
         metadata[Fields.TITLE] = None
         metadata[Fields.JOURNAL] = None
-        assert not Record.is_data_valid(metadata)
+        assert Record.missing_citation_fields(metadata) == (
+            "Citation was not provided and/or "
+            f"missing required fields to generate minimal citation: ['{Fields.CITATION}', '{Fields.TITLE}', '{Fields.JOURNAL}']."
+        )
 
     def test_is_record_creatable(self):
```

Comment:
This test should be multiple tests given the "1 reason to fail" principle

Reply:
See renamed and split tests [here](https://github.com/MITLibraries/solenoid/pull/472/commits/cf9c2a5a36a93816b9f4b205dbfa53a8e842ed92#diff-5ee692109406c92ccb8a9caf9b4e1d46a50f28cd474bae04fc9ee87862f087d4).